ckan/setup/production.ini

```diff
@@ -0,0 +1,307 @@
+
+beaker.session.type=ext:database
+beaker.session.cookie_expires=true
+beaker.session.url=%(sqlalchemy.url)s
+beaker.session.timeout=3000
+beaker.session.lock_dir=/var/tmp/ckan/lock
+
+# `paster make-config` generates a unique value for this each time it generates
+# a config file.
+app_instance_uuid = {{catalog_instance_uuid}}
+
+# repoze.who config
+who.config_file = %(here)s/who.ini
+who.log_level = warning
+who.log_file = %(cache_dir)s/who_log.ini
+# Session timeout (user logged out after period of inactivity, in seconds).
+# Inactive by default, so the session doesn't expire.
+who.timeout = 900
+# who.httponly = True # DEFAULT True already
+who.secure = True
+
+## Database Settings to be overwritten per environment
+sqlalchemy.url = postgresql://ckan:pass@db/ckan
+
+# Possible tweaks to production sizing
+# sqlalchemy.pool_size = {{ sqlalchemy_pool_size }}
+# sqlalchemy.max_overflow = {{ sqlalchemy_max_overflow }}
+
+## Site Settings
+# external url no trailing slash, to be overwritten per environment
+ckan.site_url = http://localhost:5000
+
+#ckan.use_pylons_response_cleanup_middleware = true
+
+## Authorization Settings
+
+ckan.auth.anon_create_dataset = false
+ckan.auth.create_unowned_dataset = false
+ckan.auth.create_dataset_if_not_in_organization = false
+ckan.auth.user_create_groups = false
+ckan.auth.user_create_organizations = false
+ckan.auth.user_delete_groups = false
+ckan.auth.user_delete_organizations = false
+ckan.auth.create_user_via_api = false
+ckan.auth.create_user_via_web = false # May depend on the env
+ckan.auth.roles_that_cascade_to_sub_groups = admin
+ckan.auth.public_user_details = false
+
+## Search Settings
+
+ckan.site_id = datagov_catalog
+
+solr_url = http://catalog-solr.apps.internal:8983/solr/ckan2_8
+
+#ckan.simple_search = 1
+search.facets.default = 5
+search.facets = groups tags res_format responsible-party frequency source_type
+search.facets.responsible-party.title = Organizations
+ckan.search.solr_commit = true
+
+# skip doubling package_show queries to show tracking information
+ckanext.datagovcatalog.add_packages_tracking_info = false
+
+## Redis Settings
+
+# URL to your Redis instance, including the database to be used.
+# 'rediss' is not a typo!  It specifies redis proto over TLS
+
+## Redis Settings, to be overwritten per environment
+# Set by environment
+ckan.redis.url = rediss://:REDIS_PASSWORD@REDIS_HOST:REDIS_PORT
+
+
+## CORS Settings
+
+# If cors.origin_allow_all is true, all origins are allowed.
+# If false, the cors.origin_whitelist is used.
+# ckan.cors.origin_allow_all = true
+# cors.origin_whitelist is a space separated list of allowed domains.
+# ckan.cors.origin_whitelist = http://example1.com http://example2.com
+
+
+## Plugins Settings
+
+# Note: Add ``datastore`` to enable the CKAN DataStore
+#       Add ``datapusher`` to enable DataPusher
+#		Add ``resource_proxy`` to enable resorce proxying and get around the
+#		same origin policy
+# TODO: Add saml2auth
```

Comment:
What is the error that saml2auth is giving you?

Reply:
I didn't even want to try in the initial build. This would involve setting up a new app on login.gov, secret configurations, etc...

Reply:
ah, nm as long as its in the requirements, I'm happy. I just want to make sure we know what the work is and as long as pysaml2 and saml2auth are getting built, it's unlikely we'll run into unknown errors. Don't forget to capture this as an issue 👍